### PR TITLE
add topo t0-isolated-d128u128s2

### PR DESCRIPTION
**Pull request by sdszhang**
### Description of PR

Summary:
add a new t0 topo with 128 downstream and 128 upstream ports and 2 service ports.
For bp_interfaces IP, used continuous IP from .2 to .131 to avoid going over /24 range.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

### CODE REVIEW

File: ansible/vars/topo_t0-isolated-d128u128s2.yml [hunk 1]
```diff
@@ -0,0 +1,3545 @@
...
+          64601:
+            - 10.0.0.64
+            - fc00::81
+       interfaces:
+         Loopback0:
+           ipv4: 100.1.0.33/32
+           ipv6: 2064:100::21/128
+         Ethernet1:
+           lacp: 1
+         Port-Channel1:
```
[the remark this answers is not in the document]
Comment (sdszhang) [reply ansible/vars/topo_t0-isolated-d128u128s2.yml:831]: np. 
Another question, do we need to remove these uplink ports from one_vlan_a list? only keep downstream and 2x10G links in the vlan?

[the remark this answers is not in the document]
Comment (sdszhang) [reply ansible/vars/topo_t0-isolated-d128u128s2.yml]: it's all good. removed port-channel and updated the vlan intfs too.

File: ansible/vars/topo_t0-isolated-d128u128s2.yml [hunk 2]
```diff
@@ -0,0 +1,3285 @@
...
+      Loopback0:
+        ipv4: 100.1.1.1/32
+        ipv6: 2064:100::101/128
+      Ethernet1:
+        ipv4: 10.0.2.1/31
+        ipv6: fc00::402/126
+    bp_interfaces:
+      ipv4: 10.10.246.130/24
+      ipv6: fc0a::82/64
+  ARISTA02PT1:
```
Comment (r12f) [inline ansible/vars/topo_t0-isolated-d128u128s2.yml:3267]: hi Shawn, is it possible to change this to PT0? (not PT1)

File: ansible/veos [hunk 3]
```diff
@@ -37,6 +37,7 @@ all:
           - t0-standalone-64
           - t0-standalone-128
           - t0-standalone-256
+          - t0-isolated-d128u128s2
```
Comment (r12f) [inline ansible/veos:40]: do we need to update the t0 topo list in the testbed doc with the new topo?

**Comment by r12f** [comment]
hi @sdszhang , the change looks good! I have 2 minor comments in the PR. Will sign off once they are done.

File: ansible/vars/topo_t0-isolated-d128u128s2.yml (hunk 2 above)
Comment (sdszhang) [reply ansible/vars/topo_t0-isolated-d128u128s2.yml]: changed.

**Review by r12f — APPROVED** [review]

**Comment by davidm-arista** [comment]
we are going to test this over the weekend

**Pull request merged**